### PR TITLE
More coverage for exporter

### PR DESCRIPTION
**Pull request by nfx**
make tests run faster and add more coverage

### CODE REVIEW

File: exporter/util.go [hunk 1]
```diff
@@ -142,18 +143,102 @@ func (ic *importContext) refreshMounts() error {
...
+import com.fasterxml.jackson.databind.{DeserializationFeature, ObjectMapper}
+import com.fasterxml.jackson.module.scala.experimental.ScalaObjectMapper
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+
+val readableMounts = dbutils.fs.mounts.par.map { mount =>
+try {
+	Await.result(Future {
+		dbutils.fs.ls(mount.mountPoint)
+		(mount.mountPoint
+			.replace("/mnt/", "")
```
Comment (alexott) [inline exporter/util.go:217]: it won't handle correctly the mount points that doesn't start with `/mnt`, especially built-in like, `/databricks-datasets`, etc.

```
[MountInfo(mountPoint='/databricks-datasets', source='databricks-datasets', encryptionType='sse-s3'),
 MountInfo(mountPoint='/databricks/mlflow-tracking', source='databricks/mlflow-tracking', encryptionType='sse-s3'),
 MountInfo(mountPoint='/databricks-results', source='databricks-results', encryptionType='sse-s3'),
 MountInfo(mountPoint='/databricks/mlflow-registry', source='databricks/mlflow-registry', encryptionType='sse-s3'),
 MountInfo(mountPoint='/', source='DatabricksRoot', encryptionType='sse-s3')]
```

**Review by alexott — CHANGES_REQUESTED** [review]

File: exporter/util.go (hunk 1 above)
Comment (nfx) [reply exporter/util.go:217]: but those are system-mounts, not customer-mounts

Comment (alexott) [reply exporter/util.go:217]: yep, but they are in the output of the `dbutils.fs.mounts`, so they will go into the result

Comment (nfx) [reply exporter/util.go]: `.filter(_.mountPoint.startsWith("/mnt"))` solves it :)

**Review by alexott — APPROVED** [review]
lgtm

**Pull request merged**